ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -590,3 +599,96 @@ def get_all_storageclass():
         )
     ]
     return storageclass
+
+
+def add_capacity_lso(new_node):
+    """
+    Add capacity on lso cluster
+
+    Args:
+        new_node (str): The name of the new node.
+
+    """
+    labeled_nodes = get_labeled_nodes(constants.OPERATOR_NODE_LABEL)
+    count = len(labeled_nodes)
+
+    logging.info(f"Add {new_node} to Local Volume Set")
+    path_to_nodes = "/spec/nodeSelector/nodeSelectorTerms/0/matchExpressions/0/values/-"
+    params = f"""[{{"op": "add", "path": "{path_to_nodes}", "value": "{new_node}"}}]"""
+    ocp_lvs_obj = OCP(
+        kind=constants.LOCAL_VOLUME_SET,
+        namespace=defaults.LOCAL_STORAGE_NAMESPACE,
+        resource_name=constants.LOCAL_BLOCK_RESOURCE,
+    )
+    ocp_lvs_obj.patch(params=params, format_type="json")
+
+    logging.info(f"Add {new_node} to Local Volume Discovery")
+    ocp_lvd_obj = OCP(
+        kind=constants.LOCAL_VOLUME_DISCOVERY,
+        namespace=defaults.LOCAL_STORAGE_NAMESPACE,
+    )
+    ocp_lvd_obj.patch(params=params, format_type="json")
+
+    logging.info(f"Change count field to {count} on Storage Cluster")
+    path_to_count = "/spec/storageDeviceSets/0/count"
+    params = f"""[{{"op": "replace", "path": "{path_to_count}", "value": {count}}}]"""
```

Comment:
This implies a single disk per node. Is that correct?

Reply:
Would it be better to update the storagecluster only once the LSO has created the new PV?